Core/Core/Inbox/MessageDetails/ViewModel/MessageDetailsViewModel.swift

```diff
@@ -25,6 +25,7 @@ class MessageDetailsViewModel: ObservableObject {
     @Published public private(set) var messages: [MessageViewModel] = []
     @Published public private(set) var conversations: [Conversation] = []
     @Published public private(set) var starred: Bool = false
+    @Published public private(set) var isHideReplyButton: Bool = false
```

Comment:
Same suggestion here, it's easier to under stand `if showReplyButton` than `if !isHideReplyButton`.